script.js

```diff
@@ -0,0 +1,66 @@
+// eslint-disable-next-line no-unused-vars
+/* eslint-disable no-unused-vars */
+
+const actList = document.querySelector('.act-list');
+// To toggle Nav Menu on Mobile Screen
+function clickToSwitch() {
+  const switchD = document.getElementById('switch-display');
+  if (switchD.style.display === 'block') {
+    switchD.style.display = 'none';
+  } else {
+    switchD.style.display = 'block';
+  }
+}
+
+const speakers = [
+  {
+    img: './images/features-speaker-1.png',
+    name: 'Ofentse Mokwena',
+    profession: 'Strategic Projects Lead | Uber SA',
+    description: 'Responsible for the coordination of engineering and construction work, constantly communicating with all project team.',
+  },
+  {
+    img: './images/Pamela-Mkhize.jpg',
+    name: 'Pamela Mkhize',
+    profession: 'Head of Africa Digital Solutions – Enel Group Power',
+    description: 'Shares lessons gives dynamic and educational insights on doing business in Africa',
+  },
+  {
+    img: './images/kobus_louw.jpg',
+    name: 'Kobus Louw',
+    profession: 'Founder & CEO Digemy',
+    description: 'Africa Tech Week is an unbelievable event, and it’s a great honour for me to be selected and take part in something where I can give back to a community that has supported us tremendously in the past year.',
+  },
+  {
+    img: './images/Mmapula Kgari.jpg',
+    name: 'Mmapula Kgari',
+    profession: 'Acting Executive of Broadband, Sentech',
+    description: 'Responsible for the growth and expansion plans of the business',
+  },
+  {
+    img: './images/Mlamli Booi.jpg',
+    name: 'Malamli Booi',
+    profession: 'Chief Executive Officer and Executive Director, Sentech',
+    description: 'Delegates and drives agendas, driving profitability, managing company organizational structure, strategy, and communicating with the board.',
+  },
+  {
+    img: './images/Artwell Nwaila.jpg',
+    name: 'Artwell Nwaila',
+    profession: 'Head of Creative, Google',
+    description: 'I generally develop and oversee projects at the earliest stage',
+  },
+];
```

Comment:
- [ ] Kindly make sure your **Speakers list** design sticks as much as possible to the one from the **[Behance template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)**. Right now, each speaker design misses the background image and the section title(**"Featured Speakers"**) **should have a border bottom with the correct color.** You can see the example in the images below.

| Template Design | Your Design |
| ---------------------- | ---------------------- |
| ![Screenshot from 2022-09-16 07-28-27](https://user-images.githubusercontent.com/83042742/190563415-636f430f-9480-497c-8ff1-beb7647edf7a.png) | ![Screenshot from 2022-09-17 00-18-08](https://user-images.githubusercontent.com/83042742/190815416-a769fa36-8fac-4d28-9b7e-356ca0454e83.png) |